content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -338,6 +679,142 @@ const user = await prisma.myUser.create({
 
 Learn more about this on the [Configuring your Prisma Client API](/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names) page.
 
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+As a next step, you will introspect your database. The result of the introspection will be a [data model](/concepts/components/prisma-schema/data-model) inside your Prisma schema.
+
+Run the following command to introspect your database:
+
+```terminal copy
+npx prisma db pull
+```
+
+This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a Prisma data model.
+
+After the introspection is complete, your Prisma schema file was updated:
+
+![Introspect your database with Prisma](https://res.cloudinary.com/prismaio/image/upload/v1628761155/docs/f7itiYw.png)
+
+The data model now looks similar to this (note that the fields on the models have been reordered for better readability):
+
+```prisma file=prisma/schema.prisma
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime
+  title     String   @db.VarChar(255)
+  content   String?
+  published Boolean  @default(false)
+  authorId  Int
+}
+
+model Profile {
+  id     Int     @id @default(autoincrement())
+  bio    String?
+  userId Int     @unique
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+}
+```
+
+Prisma's data model is a declarative representation of your database schema and serves as the foundation for the generated Prisma Client library. Your Prisma Client instance will expose queries that are _tailored_ to these models.
+
+You will then need to add in any missing relations between your data using [relation fields](/concepts/components/prisma-schema/relations#relation-fields):
+
+```prisma file=prisma/schema.prisma highlight=8,15;add
```

Comment:
Part of https://github.com/prisma/docs/pull/2625/files#r780407386
```suggestion
```prisma file=prisma/schema.prisma highlight=8,15,22,23;add
```